requirements.txt

```diff
@@ -1,4 +1 @@
-pytest==7.4.0
```

Comment:
We still need to pin specific dependency versions. Splitting them into a separate file is a good idea though ([example in another repo](https://github.com/closeio/tasktiger/blob/master/requirements-test.txt)).

Reply:
do you prefer to have only one `requirements-dev.txt` for all development dependencies or have different `requirements-lint.txt`, `requirements-test.text`, etc?

Reply:
I just pushed this new PR #119 creating a new `requirements-dev.txt` file